openapi/openapi.yaml

```diff
@@ -463,7 +472,7 @@ paths:
                 type: array
                 items:
                   $ref: "#/components/schemas/ToolFile"
-            text/plain:
+            application/zip:
               schema:
                 type: array
```

Comment:
The schema for zip file download is not an array, nor is it of type ToolFile.

Reply:
No way to represent it in swagger.yaml. It will convert it this way.

Reply:
Same as above